tap_github/schemas/commits.json

```diff
@@ -96,10 +96,176 @@
               "null",
               "string"
             ]
+          },
+          "contents_url": {
+            "type": [
+              "null",
+              "string"
+            ]
+          },
+          "previous_filename": {
+            "type": [
+              "null",
+              "string"
+            ]
           }
         }
       }
     },
+    "author": {
+      "type": ["null", "object"],
+      "properties": {
+        "name": {
+          "type": [
+            "null",
+            "string"
+          ]
+        },
+        "email": {
+          "type": [
+            "null",
+            "string"
+          ]
+        },
+        "login": {
+          "type": ["null", "string"]
+        },
+        "id": {
+          "type": ["null", "integer"]
+        },
+        "node_id": {
+          "type": ["null", "string"]
+        },
+        "avatar_url": {
+          "type": ["null", "string"]
+        },
+        "gravatar_id": {
```

Comment:
Use the same indentation as other fields.

Reply:
Updated